Backend/ChatAPI.Data/Models/Message.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ChatAPI.Data.Models;
+
+public class Message : BaseEntity
+{
+    //[MaxLength (500)] add fluent validation on Dbcontext
+    public string Text { get; set; }
+    public int SenderId { get; set; }
+    public int ChatRoomId { get; set; }
+
+    public List<ChatRoom> ChatRooms { get; set; }
+    public List<ChatRoomUser> ChatRoomUsers { get; set; }
```

Comment:
`public User Sender { get; set; }` is the correct definition. But User entity won't be added until to Authorization implementation, because of that remove this property from here.